packages/app-cli/app/gui/FolderListWidget.js

```diff
@@ -25,6 +26,18 @@ class FolderListWidget extends ListWidget {
 				output.push('-'.repeat(this.innerWidth));
 			} else if (item.type_ === Folder.modelType()) {
 				output.push(' '.repeat(this.folderDepth(this.folders, item.id)) + Folder.displayTitle(item));
+				if (Setting.value('showNoteCounts')) {
```

Comment:
By the way, is the folder list correctly update when the user change this setting value? using `:config showNoteCounts 0`

Reply:
Yeah, using 0 updates that setting to _false_.

Reply:
Yes it does.

https://user-images.githubusercontent.com/92958867/174653691-6198fa1a-9b4a-4b4d-97dc-424bec0af583.mov